casper/src/main/scala/coop/rchain/casper/CasperConf.scala

```diff
@@ -20,7 +20,7 @@ final case class CasperConf(
     privateKey: Option[Either[String, Path]],
     bondsFile: Option[String],
     knownValidatorsFile: Option[String],
-    numValidators: Int,
+    autogenShardSize: Int,
```

Comment:
why is `Shard`?

Reply:
This defines size of bondsfile generated that will go into genesis block's bonds map. I believe each shard should have its own bonds map in a block. Or is that network-wide table?

Reply:
I think `numActiveValidators` is more appropriate.

Reply:
`numActiveValidators ` sounds like parameter that defines the network  behaviour. But this one is just about how much identities to generate.

Reply:
Ok. I mixed them up~!